code/modules/power/singularity/particle_accelerator/particle_accelerator.dm

```diff
@@ -26,6 +26,10 @@
 	icon_state = "none"
 	anchored = 0
 	density = 1
+	obj_integrity = 500
+	max_integrity = 500

```

Comment:
Some decent hp and melee armor for balance purposes, don't want an assistant to destroy one of the unique engine parts too easily.